Add possible status values to plan/run/apply documentation (#1281)
hashicorp/terraform-website: content/source/docs/cloud/api/applies.md

An apply represents the results of applying a Terraform Run's execution plan.

> ## Attributes
> 
> ### Apply States

Reviewer: ```suggestion
## Apply State Attributes
```

> The apply state is found in `data.attributes.status`, and you can reference the following list of possible states.

Reviewer: ```suggestion
You'll find the apply state in `data.attributes.status`, as one of the following values. 
```
